dockers/docker-nat/start.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+rm -f /var/run/rsyslogd.pid
+rm -f /var/run/nat/*
+
+mkdir -p /var/warmboot/nat
+
+supervisorctl start rsyslogd
+
+supervisorctl start natmgrd
+
+sleep 5
```

Comment:
That delay was added between natsyncd and natmgrd as there were issues in natmgr's writes to APP_DB on startup, since both were writers to the nat tables in APP_DB. This 5 sec delay is now removed from start.sh, as the timing is handled in the applications.